lib/much-rails/action/base_router.rb

```diff
@@ -149,8 +149,13 @@ def delete(path, default_class_name = nil, **request_type_class_names)
 
   private
 
-  def route(http_method, path, default_class_name, **request_type_class_names)
-    url = url_class.for(self, path)
+  def route(
+        http_method,
+        url_name_or_path,
+        default_class_name,
+        **request_type_class_names)
```

Comment:
```suggestion
  def route(
        http_method,
        url_name_or_path,
        default_class_name,
        **request_type_class_names
      )
```

Reply:
@nweathersbee so method definitions are the one place we don't wrap the closing `)` down by convention. 

```ruby
def method(
      arg1,
      arg2)
  do_something
end

# vs

def method(
      arg1,
      arg2
    )
  do_something
end
```
For me, while it is inconsistent to not wrap it down to the next line, it also doesn't quite fit the same needs as doing it on method calls/etc:
 * a big improvement with wrapping on method calls is it lines up chained method calls on multiple lines nicely - there is never any chaining involved with method definitions.
 * the rule is, technically, to wrap it down to the start of the line that the opening `(` is on, doing that yields something dumb (see below). Therefore I saw it is N/A to the method definition case.

```ruby
def method(
  arg1,
  arg2
)
  do_something
end
```
Does that make sense?

/cc @jcredding